### PR TITLE
ZER-180 fix defect of cross area in calculator

### PR DESCRIPTION
**Pull request by PaulKovalyk**
dev
## JIRA

* [ZER-180](https://jira.softserve.academy/browse/ZER-180)

## Code reviewers

- [x] @loqimean 

## Summary of issue

fix defects of cross area 
Screenshot from 2022-10-31 18-24-01
image

## Summary of change

when hovering over the cross, the display of the mouse cursor changes immediately
image
changed the distance of the button relative to the top
Mobile
image
Desktop
image

## Testing approach

ToDo

## CHECK LIST
- [x]  СI passed
- [x]  Сode coverage >=95%
- [x]  PR is reviewed manually again (to make sure you have 100% ready code)
- [x]  All reviewers agreed to merge the PR
- [x]  I've checked new feature as logged in and logged out user if needed
- [x]  PR meets all conventions

### CODE REVIEW

File: app/javascript/css/calculator.css.scss [hunk 1]
```diff
@@ -141,6 +141,11 @@
   }
 }
 
+.close-img{
+  z-index: 1;
+  position: relative;
```
Comment (loqimean) [inline app/javascript/css/calculator.css.scss:146]:
```suggestion
```

File: app/views/calculators/calculator.html.slim [hunk 2]
```diff
@@ -1,7 +1,7 @@
 .jumbotron
   .container
     a.rounded.float-right.src href=root_path
-      = image_tag "close.svg", height: 40
+      = image_tag "close.svg", height: 40, class:"close-img"
```
Comment (loqimean) [inline app/views/calculators/calculator.html.slim:4]:
```suggestion
      = image_tag "close.svg", height: 40, class:"close-img position-relative"
```

**Comment by loqimean** [comment]
Screenshot 2022-11-04 at 11 57 59
space on sides shouldn't be different

File: app/javascript/css/calculator.css.scss [hunk 3]
```diff
@@ -141,6 +141,10 @@
   }
 }
 
+.close-img{
+  z-index: 1;
+}
```
Comment (loqimean) [inline app/javascript/css/calculator.css.scss:146]: Use something like this
```suggestion
.z-1 {
  z-index: 1;
}
```

File: app/views/calculators/calculator.html.slim [hunk 4]
```diff
@@ -1,7 +1,7 @@
 .jumbotron
   .container
     a.rounded.float-right.src href=root_path
-      = image_tag "close.svg", height: 40
+      = image_tag "close.svg", height: 40, class:"close-img position-relative"
```
Comment (loqimean) [inline app/views/calculators/calculator.html.slim:4]: and here
```suggestion
      = image_tag "close.svg", height: 40, class:"z-1 position-relative"
```

File: app/views/calculators/calculator.html.slim (hunk 2 above)
Comment (PaulKovalyk) [reply app/views/calculators/calculator.html.slim]: done

File: app/javascript/css/calculator.css.scss (hunk 3 above)
Comment (PaulKovalyk) [reply app/javascript/css/calculator.css.scss]: done

File: app/views/calculators/calculator.html.slim (hunk 4 above)
Comment (PaulKovalyk) [reply app/views/calculators/calculator.html.slim]: done

File: app/javascript/css/calculator.css.scss [hunk 5]
```diff
@@ -141,6 +141,10 @@
   }
 }
 
+.z-1{
```
Comment (loqimean) [inline app/javascript/css/calculator.css.scss:144]:
```suggestion
.z-1 {
```

File: app/javascript/css/calculator.css.scss [hunk 6]
```diff
@@ -154,4 +158,19 @@
   .logo-zerowaste {
     width: 140px;
   }
+  .src {
+    margin-right: 5px;
+  }
```
Comment (loqimean) [inline app/javascript/css/calculator.css.scss:163]: що це за назва?)

Comment (loqimean) [reply app/javascript/css/calculator.css.scss:163]: https://dev.to/visuellverstehen/common-mistakes-when-writing-css-with-bem-4921

**Review by loqimean — COMMENTED** [review]
So, in a nutshell, but naming is a little bit not good, so you should resolve it

File: app/javascript/css/calculator.css.scss (hunk 6 above)
Comment (PaulKovalyk) [reply app/javascript/css/calculator.css.scss]: > що це за назва?)

refactored)

**Review by loqimean — APPROVED** [review]

**Pull request merged**